lib/masamune/cached_filesystem.rb

```diff
@@ -39,11 +37,11 @@ def exists?(file)
     end
 
     def glob(file_or_glob, &block)
```

Comment:
Since you never reference the block directly, it should be removed from the parameter list.